omnigan/classifier.py

```diff
@@ -16,22 +17,24 @@ def get_classifier(opts, latent_space, verbose):
 
 
 class OmniClassifier(nn.Module):
-    def __init__(self, latent_space, loss):
+    def __init__(self, latent_space, proj_dim, loss):
         super(OmniClassifier, self).__init__()
         assert len(latent_space) == 3
         self.channels = latent_space[0]
         self.feature_size = latent_space[1]
         self.loss = loss
+        dim = proj_dim
```

Comment:
are you lazy or do you have a good reason 🛩️

Reply:
you know `cmd+D` will select next occurence and you can replace easily :p

Reply:
You already know the answer, don't you...

Reply:
I was giving you the benefit of the doubt ;)